pom.xml

```diff
@@ -41,8 +47,30 @@
             <artifactId>lombok</artifactId>
             <version>1.18.22</version>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>2.9.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>converter-gson</artifactId>
+            <version>2.9.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.kafka</groupId>
+            <artifactId>spring-kafka</artifactId>
+            <version>2.8.2</version>
+        </dependency>
```

Comment:
I included some of the required dependencies that will help the project work as described. Dependencies work by allowing you to designate external library dependencies for your project, and when you use Maven to build your project, it will fetch these libraries from the web (external repositories) and include them in your finished project. As a result, dependencies are handled automatically. These dependencies were added as a result of mistakes in the program or things I added to make the program work as it should because it lacked the appropriate dependencies. 

This link will help when adding dependencies: https://www.jetbrains.com/help/idea/work-with-maven-dependencies.html